Source/effects.cpp

```diff
@@ -737,46 +737,46 @@ TSFX sgSFX[] = {
 /*PS_WARR12*/     { sfx_STREAM | sfx_WARRIOR,  "sfx\\warrior\\warior12.wav",  nullptr },
 /*PS_WARR13*/     { sfx_WARRIOR,               "sfx\\warrior\\warior13.wav",  nullptr },
 /*PS_WARR14*/     { sfx_WARRIOR,               "sfx\\warrior\\warior14.wav",  nullptr },
-/*PS_WARR14B*/    { sfx_WARRIOR,               "sfx\\warrior\\wario14b.wav",  nullptr },
-/*PS_WARR14C*/    { sfx_WARRIOR,               "sfx\\warrior\\wario14c.wav",  nullptr },
+/*PS_WARR14B*/    { sfx_STREAM,               "sfx\\warrior\\wario14b.wav",  nullptr },
+/*PS_WARR14C*/    { sfx_STREAM,               "sfx\\warrior\\wario14c.wav",  nullptr },
 /*PS_WARR15*/     { sfx_WARRIOR,               "sfx\\warrior\\warior15.wav",  nullptr },
-/*PS_WARR15B*/    { sfx_WARRIOR,               "sfx\\warrior\\wario15b.wav",  nullptr },
-/*PS_WARR15C*/    { sfx_WARRIOR,               "sfx\\warrior\\wario15c.wav",  nullptr },
+/*PS_WARR15B*/    { sfx_STREAM,               "sfx\\warrior\\wario15b.wav",  nullptr },
+/*PS_WARR15C*/    { sfx_STREAM,               "sfx\\warrior\\wario15c.wav",  nullptr },
 /*PS_WARR16*/     { sfx_WARRIOR,               "sfx\\warrior\\warior16.wav",  nullptr },
-/*PS_WARR16B*/    { sfx_WARRIOR,               "sfx\\warrior\\wario16b.wav",  nullptr },
-/*PS_WARR16C*/    { sfx_WARRIOR,               "sfx\\warrior\\wario16c.wav",  nullptr },
-/*PS_WARR17*/     { sfx_WARRIOR,               "sfx\\warrior\\warior17.wav",  nullptr },
-/*PS_WARR18*/     { sfx_WARRIOR,               "sfx\\warrior\\warior18.wav",  nullptr },
-/*PS_WARR19*/     { sfx_WARRIOR,               "sfx\\warrior\\warior19.wav",  nullptr },
-/*PS_WARR20*/     { sfx_WARRIOR,               "sfx\\warrior\\warior20.wav",  nullptr },
-/*PS_WARR21*/     { sfx_WARRIOR,               "sfx\\warrior\\warior21.wav",  nullptr },
-/*PS_WARR22*/     { sfx_WARRIOR,               "sfx\\warrior\\warior22.wav",  nullptr },
-/*PS_WARR23*/     { sfx_WARRIOR,               "sfx\\warrior\\warior23.wav",  nullptr },
+/*PS_WARR16B*/    { sfx_STREAM,               "sfx\\warrior\\wario16b.wav",  nullptr },
+/*PS_WARR16C*/    { sfx_STREAM,               "sfx\\warrior\\wario16c.wav",  nullptr },
+/*PS_WARR17*/     { sfx_STREAM,               "sfx\\warrior\\warior17.wav",  nullptr },

```

Comment:
I don't think this is correct.
This file is "Not a chance" and it is actually used `HeroSpeech::NotAChance`